src/appleseed/renderer/kernel/lighting/sppm/sppmparameters.cpp

```diff
@@ -109,6 +109,8 @@ SPPMParameters::SPPMParameters(const ParamArray& params)
   , m_photon_tracing_rr_min_path_length(fixup_path_length(params.get_optional<size_t>("photon_tracing_rr_min_path_length", 6)))
   , m_path_tracing_max_bounces(fixup_bounces(params.get_optional<int>("path_tracing_max_bounces", -1)))
   , m_path_tracing_rr_min_path_length(fixup_path_length(params.get_optional<size_t>("path_tracing_rr_min_path_length", 6)))
+  , m_path_tracing_max_ray_intensity(params.get_optional<float>("path_tracing_max_ray_intensity", 0.0f))
+  , m_path_tracing_has_max_ray_intensity(params.get_optional<float>("path_tracing_max_ray_intensity", 0.0f) != 0.0f)
```

Comment:
Different logic than in UPT where we simply check if the `max_ray_intensity` is set or not. We must use the same logic everywhere, for our own sanity. The advantage of the logic in UPT is that one can set MRI to 0 (it's not useful in practice but there's no reason to not support this corner case if we can).

Reply:
ok